rfd/0036/services/manifest.md

```diff
@@ -216,6 +294,7 @@ Additional details are required for each type. Appropriate limits need to be ide
 Example:
 
 ```yaml
+healthchecks:
```

Comment:
@jasonpincin has done some deeper thinking about health checks. I need to integrate his work here.